WordPress/WordPressTest/ReaderTopicServiceTest.swift

```diff
@@ -138,6 +157,51 @@ final class ReaderTopicSwiftTest: XCTestCase {
 
     // MARK: Tests
 
+    /**
+    Ensure that followed sites a user unfollows from are set to unfollowed in core data when merging
+    results from the REST API.
+    */
+    func testUnfollowedSiteIsUnfollowedDuringSync() {
+        guard let context = context else {
+            XCTFail("Context is nil")
+            return
+        }
+        let remoteSites = remoteSiteInfoForTests()
+
+        // Setup
+        var expect = expectation(description: "sites saved expectation")
+        let service = ReaderTopicService(managedObjectContext: context)
```

Comment:
I guess you can remove the whole `guard` condition and force unwrap `context` here. Wdyt?

Reply:
That makes sense! Makes the test case more concise 👍

Reply:
Fixed! 6d4135c